static/docs/get-started/index.md

```diff
@@ -17,6 +17,6 @@ us a ⭐ if you like the project!
 ✅ Contribute either [on GitHub](https://github.com/iterative/dvc) or
 [on Patreon](https://www.patreon.com/DVCorg/overview) to support the project.
 
-Separate to this section, the longer [Tutorial](/doc/tutorials/deep) also
-introduces DVC step-by-step, while additionally explaining in great detail the
-motivation and what's happening internally.
+The longer [Tutorials](/doc/tutorials) section contains a step-by-step introduction to DVC.
+In addition the section explains, in great detail, the motivation behind the project and its
+internal mechanism.
```

Comment:
I like the better wording in this article but by "longer Tutorial" we specifically meant `[deep tutorial](/doc/tutorials/deep)` here. Please use that link instead 🙂

Reply:
Maybe we should actually use `[Deep Dive Tutorial](/doc/tutorials/deep)` AND update the title to **Deep Dive Tutorial** in /doc/tutorials/deep (which currently only reads "**Tutorial**".

Reply:
I just realized #739 states to change this link. OK, never mind! (I'll address the doc title separately.)